pkg/oc/cli/admin/prune/imageprune/helper.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/docker/distribution/registry/api/errcode"
 	"github.com/golang/glog"
 
+	corev1 "k8s.io/api/core/v1"
```

Comment:
This might conflict with @soltysh 's [current PR](https://github.com/openshift/origin/pull/20532) :)